go.mod

```diff
@@ -15,10 +15,10 @@ require (
 )
 
 require (
-	github.com/NearNodeFlash/lustre-fs-operator v0.0.1-0.20231031201943-531116c1194e
-	github.com/NearNodeFlash/nnf-dm v0.0.1-0.20231031212032-66e6f08012fc
+	github.com/NearNodeFlash/lustre-fs-operator v0.0.1-0.20231115220321-eb84599f9630
+	github.com/NearNodeFlash/nnf-dm v0.0.6
```

Comment:
You wanted to pin to a release?

Reply:
No, not sure what happened there, but it's fixed now.